src/screens/mentee-home/components/notifications/notifications.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```

Comment:
Let's rename it into notifications-section, because this component does not represent the notifications page